MinecraftClient/Settings.cs

```diff
@@ -230,6 +231,7 @@ public static void LoadSettings(string settingsfile)
                                                 case "privatemsgscmdname": PrivateMsgsCmdName = argValue.ToLower().Trim(); break;
                                                 case "botmessagedelay": botMessageDelay = TimeSpan.FromSeconds(str2int(argValue)); break;
                                                 case "debugmessages": DebugMessages = str2bool(argValue); break;
+                                                case "autoattackmobs": AutoAttackMobs = str2bool(argValue); break;
```

Comment:
If implemented as an internal ChatBot, this setting would be moved to its own INI file section (same with autofish 😉)